src/Reflection/ReflectionParameter.php

```diff
@@ -490,7 +490,10 @@ public function getClass() : ?ReflectionClass
             ));
         }
 
-        return $this->reflector->reflect($className);
+        /** @var ReflectionClass $reflectionClass */
```

Comment:
With the above ClassReflector change, you could instead change the `@var` to `@var ClassReflector $reflector`